.travis.yml

```diff
@@ -27,71 +23,67 @@ install:
-    # linux publishable node v4/release
-    - os: linux
-      env: BUILDTYPE=release
-      node_js: 4
-    # linux publishable node v4/debug
+
+    ## ** Builds that are published **
+
+    # linux cfi build node v6/release
     - os: linux
-      env: BUILDTYPE=debug
-      node_js: 4
-    # linux publishable node v6
+      env: BUILDTYPE=release TOOLSET=cfi CXXFLAGS="-fsanitize=cfi -fvisibility=hidden" LDFLAGS="-fsanitize=cfi"
+      node_js: 6
+    # linux publishable node v6/release
     - os: linux
       env: BUILDTYPE=release
       node_js: 6
     # linux publishable node v6/debug
     - os: linux
       env: BUILDTYPE=debug
       node_js: 6
-    # osx publishable node v4
+    # osx publishable node v6/release
     - os: osx
-      osx_image: xcode8.2
+      osx_image: xcode9.2
       env: BUILDTYPE=release
-      node_js: 4
-    # osx publishable node v6
+      node_js: 6
+    # osx publishable node v6/debug
     - os: osx
-      osx_image: xcode8.2
+      osx_image: xcode9.2
       env: BUILDTYPE=release
       node_js: 6
-    # Sanitizer build node v4/Debug
+    # linux sanitizer build node v6/debug
     - os: linux
       env: BUILDTYPE=debug TOOLSET=asan
-      node_js: 4
+      node_js: 6
       sudo: required
       # Overrides `install` to set up custom asan flags
       install:
-        - ./scripts/setup.sh --config local.env
-        # put mason and clang++ on PATH
-        - source local.env
-        # Note: to build without stopping on errors remove the -fno-sanitize-recover=all flag
-        # You might want to do this if there are multiple errors and you want to see them all before fixing
-        - export CXXFLAGS="${MASON_SANITIZE_CXXFLAGS} -fno-sanitize-recover=all"
-        - export LDFLAGS="${MASON_SANITIZE_LDFLAGS}"
-        - make ${BUILDTYPE}
-      # Overrides `script` to disable asan LD_PRELOAD before publishing
+        - make sanitize
+      # Overrides `before_script` (tests are already run in `make sanitize`)
       before_script:
-        - export LD_PRELOAD=${MASON_LLVM_RT_PRELOAD}
-        - export ASAN_OPTIONS=fast_unwind_on_malloc=0:${ASAN_OPTIONS}
-        - npm test
-        - unset LD_PRELOAD
-        # after successful tests, publish binaries if specified in commit message
-        - ./scripts/publish.sh --toolset=${TOOLSET:-} --debug=$([ "${BUILDTYPE}" == 'debug' ] && echo "true" || echo "false")
+    # osx sanitizer build node v6/debug
+    - os: osx
+      env: BUILDTYPE=debug TOOLSET=asan
+      node_js: 6
+      sudo: required
+      # Overrides `install` to set up custom asan flags
+      install:
+        - make sanitize
+      # Overrides `before_script` (tests are already run in `make sanitize`)
+      before_script:
+
+    ## ** Builds that do not get published **
+    
     # g++ build (default builds all use clang++)
     - os: linux
-      env: BUILDTYPE=debug CXX="g++-6" CC="gcc-6"
-      node_js: 4
+      env: BUILDTYPE=debug CXX="g++-6" CC="gcc-6" LINK="g++-6" AR="ar" NM="nm"
```

Comment:
Mention this in the docs if someone wanted to use gcc instead?

Reply:
Done in https://github.com/mapbox/node-cpp-skel/pull/111/commits/69639045fb7c7f6396de12544e06575805c49900 @GretaCB